backend/server_test.go

```diff
@@ -0,0 +1,293 @@
+package backend
+
+import (
+	"fmt"
+	"github.com/gomodule/redigo/redis"
+	"github.com/stretchr/testify/require"
+	"log"
+	"net/http"
+	"net/http/httptest"
+	"strconv"
+	"testing"
+)
+
+
+var people = []Person{
+	{
+	"horsey",
+	42,
+	},
+	{
+	"small bear",
+	55,
+	},
+	{
+		"doggo",
+		12,
+	},
+}
+const defaultOffset = 5.0
+
+func setup() *App {
+	app := &App{}
+	app.init("TESTING")
+	return app
+}
+
+func initDB(app *App, people []Person) error {
+	conn := app.pool.Get()
+
+	for _, person := range people {
+		_, err := conn.Do("HSET", person.Name, "name", person.Name, "score", person.Score)
+		if err != nil {
+			cleanDB(app)
+			return err
+		}
+	}
+	return nil
+}
+
+func cleanDB(app *App) {
+	conn := app.pool.Get()
+	_, err := conn.Do("FLUSHDB")
+	if err != nil {
+		log.Println(err)
+	}
+}
+
+func createRequestWithQuery(endpoint string, query map[string]string) (*http.Request, error) {
+	req, err := http.NewRequest("GET", endpoint, nil)
+	if err != nil {
+		return &http.Request{}, err
+	}
+
+	q := req.URL.Query()
+	for key, val := range query {
+		q.Add(key, val)
+	}
+	req.URL.RawQuery = q.Encode()
+	return req, nil
+}
+
+func TestAddPeopleSuccess(t *testing.T) {
+	app := setup()
+	defer cleanDB(app)
+
+	testCasesSuccess := map[string]struct {
+		name 	string
+		score	string
+
+		expectedCode	int
+		expectedScore	float64
+	}{
+		"success below 100": {
+			"horsey",
+			"99",
+			http.StatusFound,
+			99,
+		},
+		"success below 199": {
```

Comment:
Curious as to why you chose 199 shouldn't all values below 100 behave the same? One case you might want to handle is the max/min of your int (maybe have a check in your server.go to make sure that initial and byPoints aren't greater than the max/min of an int